python-threatexchange/threatexchange/TE.py

```diff
@@ -180,9 +180,9 @@ def getInfoForIDs(self, ids, **kwargs):
     # ----------------------------------------------------------------
     # Gets threat updates for the given privacy group.
     @classmethod
-    def getThreatUpdates(self, privacy_group, **kwargs):
-        if "next" in kwargs:
-            url = kwargs["next"]
+    def getThreatUpdates(self, privacy_group, next_page=None, **kwargs):
```

Comment:
(not blocking): I generally recommend against passing **kwargs to functions, since it erases type information, and makes it hard to discovered when you mistype an argument name